capsules/classifier_safety_gear_openvino/backend.py

```diff
@@ -0,0 +1,72 @@
+from typing import Dict, List
+
+import numpy as np
+
+from vcap import DETECTION_NODE_TYPE, OPTION_TYPE, BaseStreamState
+
+from vcap_utils import BaseOpenVINOBackend, linear_assignment, iou_cost_matrix
+
+from .config import safety_hat, safety_vest, gear_types
+
+from . import config
+
+
+class Backend(BaseOpenVINOBackend):
+    label_map: Dict[int, str] = {1: "unknown", 2: safety_vest,
+                                 3: "unknown", 4: safety_hat}
+
+    def process_frame(self, frame: np.ndarray,
+                      detection_nodes: DETECTION_NODE_TYPE,
+                      options: Dict[str, OPTION_TYPE],
+                      state: BaseStreamState) -> DETECTION_NODE_TYPE:
+        if len(detection_nodes) == 0:
+            return detection_nodes
+
+        confidence_threshold = options[config.confidence_threshold]
+
+        input_dict, resize = self.prepare_inputs(frame)
+        prediction = self.send_to_batch(input_dict).get()
```

Comment:
```suggestion
        prediction = self.send_to_batch(input_dict).result()
```

Alex just recently changed this system to use futures instead of queues.

Reply:
vcap-0.3 is not going to work with BrainFrame 0.27. And I'm merging this branch to vcap-v0.2 branch. So I think I shouldn't bump up the version at this time. Maybe we should do it when we want to merge it into vcap-v0.3 branch.